src/pages/resultsView/enrichments/AlterationEnrichmentsTable.tsx

```diff
@@ -24,6 +24,7 @@ export interface IAlterationEnrichmentTableProps {
     onCheckGene?: (hugoGeneSymbol: string) => void;
     onGeneNameClick?: (hugoGeneSymbol: string) => void;
     checkedGenes?: string[];
+    groupsCount: number;
```

Comment:
I think this can be derived from `props.data` as `_.values(props.data[0].groupsSet).length`